README.md

```diff
@@ -4,7 +4,7 @@ CascadeBot runs on JDK 11 to have the latest features available to provide the b
 
 ### Project Owners
 
-Lots of talented developers have put work into CascadeBot, although the three main owners have put incredible amounts of work into CascadeBot, getting it to the stage it is now. Here's some more information about them:
+Lots of talented developers have put work into CascadeBot, although the three main owners have put incredible amounts of work into it, getting it to the stage it is now. Here's some more information about them:
```

Comment:
I prefer it saying cascade bot instead of `it`

Reply:
Actually i believe i said this to you before and you still changed it :P, but @mrjvs and @binaryoverload should also give their opinions (especially binary cause he's the writing guy)

Reply:
Tbh I'm not a huge fan of this entire sentence, I'm gonna come up with something else

Reply:
If you could let me know when you've come up with something else @binaryoverload